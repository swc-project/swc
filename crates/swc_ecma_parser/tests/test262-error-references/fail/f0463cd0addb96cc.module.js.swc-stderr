
  x Unexpected token `;`. Expected a string literal
   ,-[$DIR/tests/test262-parser/fail/f0463cd0addb96cc.module.js:1:1]
 1 | import {} from;
   :               ^
   `----
