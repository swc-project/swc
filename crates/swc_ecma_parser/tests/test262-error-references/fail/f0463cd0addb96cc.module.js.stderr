
  x Unexpected token `;`. Expected a string literal
   ,-[$DIR/tests/test262-parser/fail/f0463cd0addb96cc.module.js:2:1]
 2 | import {} from;
   :               ^
   `----
